examples/output/main.go

```diff
@@ -24,15 +24,13 @@ func main() {
 		log.Fatalf("unable to connect: %v", err)
 	}
 	defer c.Close()
-	outputClient, err := c.Output()
+	outputsClient, err := c.Outputs()
 	if err != nil {
 		log.Fatalf("unable to obtain an output client: %v", err)
 	}
 
 	ctx := context.Background()
-	// Keepalive true means that the client will wait indefinitely for new events to come
-	// Use keepalive false if you only want to receive the accumulated events and stop
-	fcs, err := outputClient.Subscribe(ctx, &output.Request{Keepalive: true})
+	fcs, err := outputsClient.Get(ctx, &outputs.Request{})
```

Comment:
This breaks the example, because it won't keep the connection open. The `fcs.Recv()` below relies in this connection open, so it will only get the events kept in memory by the Falco daemon and exiting in the next iteration of the `for` loop (because the connection is now closed).

Reply:
Hello @bgarber this is the intended use case of the `falco.outputs.service/get` RPC.

The connection gets opened, then all the Falco outputs in the stream are received until there are no more events, and finally, the connection gets closed.

Reply:
@bgarber thanks for checking this out! Yes we changed the way outputs works by splitting into two different rpc calls:

- `get` connects, gets all the events available and then closes
- `Sub` is the same as the old `Subscribe`, connects, gets all the events and then keeps getting them until something stops the process

If you want to see an example with the old behavior look at:
- `examples/output_unix_socket_bidi`
- `examples/output_bidi`

Reply:
@fntlnz Thank you very much for clarifying this for me!

Reply:
Thanks to you too @bgarber